tensorflow_toolkit/text_recognition/README.md

```diff
@@ -7,16 +7,17 @@ Models code is designed to enable export to frozen graph and inference on CPU vi
 
 ### Prerequisites
 
-* Ubuntu\* 16.04
-* Python\* 3.5.2
-* TensorFlow\* 1.13
+* Ubuntu\* 16.04-20.04
+* Python\* 3.5-3.7 (Python >=3.8 is not supported by TensorFlow 1.x)
+* TensorFlow\* 1.13-1.15
 * CUDA\* 10.0
+* cuDNN\* 7.6.5
 
 ### Installation
 
 1. Create virtual environment:
     ```bash
-    virtualenv venv -p python3 --prompt="(tr)"
+    virtualenv venv -p python3.7 --prompt="(tr)"
```

Comment:
IMO, it's better to either leave python3 here, or remove other Python versions from a list of supported ones in the Prerequisites section.